src/components/charts/largeScaleStructurePlot/2DChart.jsx

```diff
@@ -0,0 +1,302 @@
+      color = '#A8D0E6';
+    } else {
+      string =
+        dat.RA.toFixed(2) +
+        ', ' +
+        dat.Dec.toFixed(2) +
+        ', ' +
+        dat.redshift.toFixed(2);
+      color = '#374785';
+    }
+    return this.formatTooltipPoint(color, string);
+  };
+
+  getLegend = () => {
+    const { selectedPt } = this.state;
+    let legendPt = '';
+    let legendIcon = 'none';
+    if (selectedPt.length !== 0) {
+      legendPt = 'Selected Data Point';
+      legendIcon = 'circle';
+    }
+    return {
+      orient: 'vertical',
+      right: 5,
+      data: [
+        {
+          name: 'Data Point',
+          icon: 'circle',
+        },
+        {
+          name: legendPt,
+          icon: legendIcon,
+        },
+      ],
+    };
+  };
+
+  getAxisInfo(title) {
+    return {
+      nameTextStyle: {
+        fontSize: 20,
+        fontWeight: 'bold',
+      },
+      name: title,
+      nameLocation: 'middle',
+      nameGap: 25,
+      axisLine: {
+        lineStyle: {
+          color: 'black',
+          opacity: 1,
+        },
+      },
+    };
+  }
+
+  clearArray() {
+    this.setState(prevState => ({
+      ...prevState,
+      selectedPt: [],
+    }));
+  }
+
+  getOption() {
+    const {
+      data,
+      selectedData,
+      // highlightSeries,
+      largeBool,
+      trimBool,
+      // sliderChange,
+      // sliderFunc,
+    } = this.props;
+    const { selectedPt } = this.state;
+
+    let opacityNum = 1;
+    let ptColor = '#A8D0E6';
+    if (!trimBool) {
+      opacityNum = 0.25;
+      ptColor = '#374785';
+    }
+    // if (sliderChange) {
+    //   this.clearArray();
+    //   sliderFunc(false);
+    // }
+
+    if (data) {
+      if (Object.keys(data).length === 0) return {};
+      if (data.length === 0) return {};
+    }
+
+    // if (trimBool && highlightSeries.length !== 0) {
+    //   data = [];
+    // }
+
+    return {
+      // title: {
+      //   text: 'Exploring Observable Universe',
+      // },
+      legend: this.getLegend(),
+      dataset: {
+        source: selectedData,
+        dimensions: ['RA', 'Dec', 'redshift'],
+      },
+      tooltip: {
+        borderColor: '#777',
+        borderWidth: 2,
+        formatter: this.getTooltipFormat,
+      },
+      xAxis: this.getAxisInfo('RA'),
+      yAxis: this.getAxisInfo('Dec'),
+      visualMap: [
+        {
+          show: false,
+          type: 'piecewise',
+          dimension: 2,
+          pieces: [
+            { min: 0, max: 0.09, color: '#ffbaba' },
+            { min: 0.09, max: 0.15, color: '#ff5252' },
+            { min: 0.15, max: 0.2, color: ' #a70000' },
+          ],
+          outOfRange: {
+            color: ['#000'],
+          },
+        },
+      ],
+
+      color: ['#A8D0E6', '#54ff', '#374785'],
+      series: [
+        {
+          name: 'Data Point',
+          symbolSize: 8,
+          type: 'scatter',
+          legendHoverLink: true,
+          large: largeBool,
+          data,
```

Comment:
Axes of graph read Dec and RA, which is, I think, the dimensions we are interested in showing here, however the values don't line up correctly: this graph is definitely plotting RA & Redshift.  ?this is probably related to the fact that the data coming into this component is an array of arrays now whereas the work you copied and pasted this out of was maybe accepting an array of objects before?  I'm not sure.  I think there's a dimensions filed you can use to point the graph to the keys/indexes you want it to plot (I think this is why you see RA and DEC pop up in the dimensions for the dataset option above?  Although this is maybe the syntax you would use to reference keys in an object and so maybe is not doing anything here? ) And in fact I just commented that line out and see no visual or behavioral change so I think what values are getting plotted right now is 100% running off of whatever eCharts grabs by default, and that's not quite lining up with what we have.  I just tried updating the `arrayifyData` method in the container to spit out the values into arrays in a different order, and that solved the issue in the2D plot but messed it up in the 3D plot.  So either here or there something needs to change to better define what values get used where.  And either way I'd like the code to be explicit about what values to use (i.e. if given the option I'd rather see us telling echarts which dimensions of the data to plot rather than changing our data to line up to echarts' default behavior).